osh/cmd_eval.py

```diff
@@ -1046,7 +1046,7 @@ def _Dispatch(self, node, cmd_st):
           else:
             # Only print warnings, never fatal.
             # Bash oddly only exits 1 for 'return', but no other shell does.
-            self.errfmt.PrefixPrint(msg, prefix='warning: ', span_id=tok.span_id)
+            self.errfmt.PrefixPrint(msg, 'warning: ', tok)
```

Comment:
Here we do NOT pass `loc.Span()` -- we just pass the token itself.

Remember that a Token **is a** location

Anywhere you would se `loc.Span(tok.span_id)` -- that's wrong and should just be `tok`

That's like doing a "double conversion" and is not necessary